messages/JsonMessages/PlayerTextures.ts

```diff
@@ -7,18 +8,18 @@ import { z } from "zod";
 
 //The list of all the player textures, both the default models and the partial textures used for customization
 
-const wokaTexture = z.object({
-    id: z.string(),
-    name: z.string(),
-    url: z.string(),
-    tags: z.array(z.string()).optional(),
-    tintable: z.boolean().optional(),
+export const wokaTexture = z.object({
+    id: extendApi(z.string(), {description: 'The identifier of the texture in the database.', example: '03395306-5dee-4b16-a034-36f2c5f2324a'}),
```

Comment:
```suggestion
    id: extendApi(z.string(), {description: 'A unique identifier for this texture.', example: '03395306-5dee-4b16-a034-36f2c5f2324a'}),
```

We should not mention a database as there might be no database.